applications/FluidDynamicsApplication/tests/cpp_tests/test_symbolic_stokes_element.cpp

```diff
@@ -0,0 +1,328 @@
+            model_part.AddNodalSolutionStepVariable(DYNAMIC_VISCOSITY);
+            model_part.AddNodalSolutionStepVariable(DYNAMIC_TAU);
+            model_part.AddNodalSolutionStepVariable(PRESSURE);
+            model_part.AddNodalSolutionStepVariable(VELOCITY);
+
+            // Process info creation
+            double delta_time = 0.1;
+            ProcessInfo &process_info = model_part.GetProcessInfo();
+            process_info.SetValue(STEP, 0);
+            process_info.SetValue(DYNAMIC_TAU, 0.0);
+            process_info.SetValue(DELTA_TIME, delta_time);
+            process_info.SetValue(START_TIME, 0.0);
+            model_part.CloneTimeStep(delta_time);
+            process_info[STEP] = 1;
+            TimeDiscretization::BDF time_disc_BDF2(2);
+            time_disc_BDF2.ComputeAndSaveBDFCoefficients(process_info);
+
+            // Set element properties
+            Properties::Pointer p_elem_prop = model_part.CreateNewProperties(0);
+            p_elem_prop->SetValue(DENSITY, 1000.0);
+            p_elem_prop->SetValue(DYNAMIC_VISCOSITY, 1000.0);
+            Newtonian3DLaw::Pointer p_const_law(new Newtonian3DLaw());
+            p_elem_prop->SetValue(CONSTITUTIVE_LAW, p_const_law);
+
+            // Geometry creation
+            model_part.CreateNewNode(1, 0.0, 0.0, 0.0);
+            model_part.CreateNewNode(2, 1.0, 0.0, 0.0);
+            model_part.CreateNewNode(3, 1.0, 1.0, 0.0);
+            model_part.CreateNewNode(4, 0.0, 1.0, 0.0);
+            model_part.CreateNewNode(5, 0.0, 0.0, 1.0);
+            model_part.CreateNewNode(6, 1.0, 0.0, 1.0);
+            model_part.CreateNewNode(7, 1.0, 1.0, 1.0);
+            model_part.CreateNewNode(8, 0.0, 1.0, 1.0);
+            model_part.CreateNewElement("SymbolicStokes3D8N", 1, {1, 2, 3, 4, 5, 6, 7, 8}, p_elem_prop);
+
+            // Set the nodal DENSITY and DYNAMICS_VISCOSITY
+            Element::Pointer p_element = model_part.pGetElement(1);
+            for (NodeIteratorType it_node = model_part.NodesBegin(); it_node != model_part.NodesEnd(); ++it_node)
+            {
+                it_node->FastGetSolutionStepValue(DENSITY) = p_elem_prop->GetValue(DENSITY);
+                it_node->FastGetSolutionStepValue(DYNAMIC_VISCOSITY) = p_elem_prop->GetValue(DYNAMIC_VISCOSITY);
+                if (it_node->Z() == 0.0)
+                {
+                    it_node->Fix(VELOCITY_X);
+                    it_node->Fix(VELOCITY_Y);
+                    it_node->Fix(VELOCITY_Z);
+                }
+                if (it_node->Z() == 1.0)
+                {
+                    it_node->FastGetSolutionStepValue(VELOCITY_X) = 0.1;
+                    it_node->FastGetSolutionStepValue(VELOCITY_Y) = 0.1;
+                    it_node->FastGetSolutionStepValue(VELOCITY_Z) = 0.1;
+                }
+            }
+
+            // Compute RHS and LHS
+            Vector RHS = ZeroVector(32);
+            Matrix LHS = ZeroMatrix(32, 32);
+            p_element->Initialize();

```

Comment:
```suggestion
            p_element->Initialize(process_info);
```

you were using the deprecated interface

Reply:
done.